### ISSUE TITLE
New Rule: no-empty-string-values

### ISSUE THREAD

**Issue by dananaprey** [eslint/json#54]
### Rule details

Detect empty string values in JSON

### What type of rule is this?

Suggests an alternate way of doing something

### Example code

```jsonc
Real life example:

I have a JSON file for application localization
{
  "localizationKey1": "Some translation 1",
  "localizationKey2": "",
  "localizationKey3": "Some translation 3",
  ...
}

I want to prevent empty translations values ​​to avoid missing text in the application

This rule should help
```

### Participation

- [x] I am willing to submit a pull request to implement this rule.

### Additional comments

Similar rules, if necessary, can be made for values **{}, [], null**

I am also willing to submit a pull requests to implement this rules

**Comment by nzakas** [comment]
Similar to #55, I'm not sure if this is generic enough to be included in the plugin. Empty string values are completely valid and I'm not sure represent an error in most cases.

@eslint/eslint-team what do you think?

**Comment by mdjermanovic** [comment]
I agree that empty string values are generally valid so this doesn't seem generic enough to be included in the plugin.
